### PR TITLE
WT-13318 Moving bash script to separate files - tests

### PR DESCRIPTION
**Pull request by bzm-jas**

### CODE REVIEW

File: test/schema_abort_predictable.sh [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+    echo "Current args: $@"
+    exit 1
+fi
+
+times=$1
+
+runtime=20  # seconds
+nthreads=5
+
+toolsdir=../../../../tools
```
Comment (ajmorton) [inline test/schema_abort_predictable.sh:19]: These hardcoded paths assume the script is run from `wiredtiger/cmake_build/test/csuite/schema_abort` which we set with `working_dir` in the evergreen task. 
Now this code is in a script that requirement is less obvious. Can we have a naiive check:
```
if [ "basename $(pwd)" != "schema_abort" ];
    echo "This script must be run from the build/test/csuite/schema_abort folder"
    exit 1;
fi
```
?

Same comment for the hardcoded paths in `checkpoint_test_predictable`

**Review by ajmorton — APPROVED** [review]
LGTM with some nits!

File: test/format_test_predictable.sh [hunk 2]
```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+
+if [ $# -lt 1 ]; then
+    echo "Error: Not enough arguments given."
+    echo "Current args: $@"
+    exit 1
+fi
```
[the remark this answers is not in the document]
Comment (bzm-jas) [reply test/format_test_predictable.sh]: good idea, added in [6299b97](https://github.com/wiredtiger/wiredtiger/pull/10921/commits/6299b97bfd7d576aa51b350f8eede428784cd1f5)

File: test/format_test_predictable.sh [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+
+times=$1
+test_format_extra_args=$2
+
+# Get a random value with leading zeroes removed, /bin/sh version.
+rando() {
+  tr -cd 0-9 </dev/urandom | head -c 5 | sed -e 's/0*\(.\)/\1/'
+}
+# Fail, showing the configuration file.
```
[the remark this answers is not in the document]
Comment (bzm-jas) [reply test/format_test_predictable.sh]: fixed here [6299b97](https://github.com/wiredtiger/wiredtiger/pull/10921/commits/6299b97bfd7d576aa51b350f8eede428784cd1f5)

File: test/schema_abort_predictable.sh (hunk 1 above)
Comment (bzm-jas) [reply test/schema_abort_predictable.sh]: check added in [6299b97](https://github.com/wiredtiger/wiredtiger/pull/10921/commits/6299b97bfd7d576aa51b350f8eede428784cd1f5)

File: test/evergreen.yml [hunk 4]
```diff
@@ -892,46 +892,8 @@ functions:
         set -o errexit
         set -o verbose
```
[the remark this answers is not in the document]
Comment (bzm-jas) [reply test/evergreen.yml:882]: I did some research but am getting mixed infos, so I'm setting them again inside the script to be safe, see: [032c0b1](https://github.com/wiredtiger/wiredtiger/pull/10921/commits/032c0b156228ced54883b7ef07864d8259e19e26)

**Review by jiechenbo — APPROVED** [review]
LGTM!

**Review by ajmorton — APPROVED** [review]
LGTM! One last question, but again feel free to resolve without re-review

File: test/evergreen/cppsuite_test_run.sh [hunk 5]
```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+set -o verbose
+
+if [[ $# -lt 1 || $# -gt 3 ]]; then
```
[the remark this answers is not in the document]
Comment (bzm-jas) [reply test/evergreen/cppsuite_test_run.sh:5]: yes this check here also accommodates for the tests that don't need configs

**Pull request merged**